app/models/ConferenceDescriptor.scala

```diff
@@ -214,8 +214,8 @@ object ConferenceDescriptor {
   // TODO The idea here is to describe in term of Agenda, for each rooms, the slots. This is required only for the Scheduler
   object ConferenceSlots {
 
-    val firstDay = "2017-04-10"
-    val secondDay = "2017-04-11"
+    val firstDay = "2018-04-10"
```

Comment:
I don't know Scala so I won'r risk doing a review, however I think it should be 2018-04-23 and 2018-04-24 here.

Reply:
Good catch